organizers.md

```diff
@@ -1,12 +1,11 @@
 ### General Co-Chairs
-- Jeronimo Castrillon, TU Dresden, Germany
 - Hokeun Kim, Arizona State University, USA
+- Jeronimo Castrillon, TU Dresden, Germany
 
 ### Organizing Committee
-- Jeronimo Castrillon, TU Dresden, Germany
```

Comment:
Should Jeronimo and Hoekun be listed in the organizing committee when they are the general co-chairs?